docs/user-guide/media-editor-window.md

```diff
@@ -45,16 +45,44 @@ With a click on **Save** <img src="../img/button/save-icon.jpg" class="ms-docbut
 
 Once selectd in the list, the image can be included in the story by clicking on the **Apply** button <img src="../img/button/apply-button2.jpg" class="ms-docbutton"/>.
 
+## Videos
+
+In order to add a video, the stroy editor can click on the **Video** tab <img src="../img/button/videos.jpg" class="ms-docbutton"/> in order to switch to the videos section. In this section of the media editor window, with a click on the **Add** button <img src="../img/button/+++.jpg" class="ms-docbutton"/>, it is possible to define the video settings.
+
+<img src="../img/media-editor-window/video-settings.jpg" class="ms-docimage"/>
+
+In particular, it is possible to insert the following parameters:
+
+* The **Video URL** of the video
+
+* The **Title** of the video
+
+* A **Description**, used to explain the contents of the video. The description is available in the viedos preview list and, if the video is added as content in the *Pharagraph section*, in the *Immersive section* or in the *Media section*, under the video
```

Comment:
* A **Description**, used to explain the contents of the video. The description is available in the viedos preview list and it can appear under the video itself if the video is added as a content of a section (*Pharagraph section*, *Immersive section* or *Media section*)